test/Riok.Mapperly.IntegrationTests/Models/TestObjectProjection.cs

```diff
@@ -5,9 +5,11 @@ namespace Riok.Mapperly.IntegrationTests.Models
 {
     public class TestObjectProjection
     {
+        public int Id { get; set; }
+
         public int CtorValue { get; set; }
 
-        public int IntValue { get; set; }
+        public int Intvalue { get; set; }
```

Comment:
This can be removed?

Reply:
Fixed the typo but keep it to ensure it is correctly mapped.